src/types/FieldMap.ts

```diff
@@ -9,7 +9,7 @@ export interface FieldMapOption {
 
 export interface FieldMap {
   name: string;
-  type?: 'ID' | 'COMPOSITE' | 'TO_ONE' | 'TO_MANY' | 'SCALAR';
+  type?: 'ID' | 'COMPOSITE' | 'TO_ONE' | 'TO_MANY' | 'SCALAR' | 'date';
```

Comment:
Is this LowerCase in the API, seems like they made an error there?
If it is not fixable we should look at making a method to check this that is case-insensitive.

Reply:
I really just added this to fix some build issues I was seeing. Several folks have set this type field to 'date' in the app. I think I will actually pick/rebase of your branch, since your changes will fix my build issues. Date fields in the API are type 'SCALAR', so I do think this was an error on their part.